flake.nix

```diff
@@ -51,7 +51,7 @@
               cargoLock = {
                 lockFile = "${self}/Cargo.lock";
                 outputHashes = {
-                  "dag-0.1.0" = "sha256-ZSPdQ5WLx977p36bIbxhbsp2k57DbFaAemGj5mcKavw=";
+                  "renderdag-0.1.0" = "sha256-ZSPdQ5WLx977p36bIbxhbsp2k57DbFaAemGj5mcKavw=";
```

Comment:
How do I figure out what hash to use here?

Reply:
You can just delete it (see other comment). I that wasn't the case the failed CI output would tell you what the expected hash would have been.

Looking at the CI is the easiest approach if you're not using Nix. Otherwise you can get the same information from `nix build`.